Cargo.toml

```diff
@@ -26,3 +26,6 @@ std = ["rustc-hex"]
 [[example]]
 name = "modular"
 required-features = ["std"]
+
+[badges]
+maintenance = { status = "deprecated" }
```

Comment:
I wonder if I need to bump the version to publish this?

Reply:
yes